package.json

```diff
@@ -143,5 +143,8 @@
         "dist/react-map-gl.esm.js"
       ]
     }
-  ]
+  ],
+  "dependencies": {
+    "@mapbox/point-geometry": "^0.1.0"
```

Comment:
Hi @cazzer! What was the issue that needed to add `@mapbox/point-geometry` as a dependency?

Reply:
I got the following ESLint error when trying to commit:
```
/Users/calebbrewer/dev/cazzer/react-map-gl/src/components/Marker/index.js
  8:32  error  '@mapbox/point-geometry' should be listed in the project's dependencies. Run 'npm i -S @mapbox/point-geometry' to add it  import/no-extraneous-dependencies
```
I can remove it, but the build will fail.